public/demo-image/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://raw.githubusercontent.com/revisit-studies/study/v1.0.5/src/parser/StudyConfigSchema.json",
+  "$schema": "https://raw.githubusercontent.com/revisit-studies/study/time-to-next/src/parser/StudyConfigSchema.json",
```

Comment:
Confused what the change is for? Just for testing the schema validation?

Reply:
This lets the tests pass here, not just locally. It uses the updated schema